.ci/install_kata.sh

```diff
@@ -9,24 +9,32 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+local_chronic() {
+	if [ -n "$CI" ]; then
```

Comment:
shouldn't this be `-z`

Reply:
or maybe check  for` [ "$CI" == true ]` ?

Reply:
It should be `-z`, yep - mea culpa.

I did do a search to see if we had a common idiom to check the `CI` var:

```bash
$  git grep '"$CI"' | fgrep if
.ci/go-test.sh:         if [ "$CI" = true ] && [ -n "$KATA_DEV_MODE" ]; then
.ci/go-test.sh: if [ "$CI" = true ] || [ -n "$KATA_DEV_MODE" ]; then
.ci/install_cri_containerd.sh:if [ -n "$CI" ]; then
.ci/install_kata.sh:    if [ -n "$CI" ]; then
.ci/install_openshift.sh:if [ "$ID" != "fedora" ] && [ "$CI" == true ]; then
.ci/setup.sh:   if ! sudo docker version | grep -q "$docker_version" && [ "$CI" == true ]; then
.ci/setup.sh:           if [ "$CI" == true ] && grep -q "N" /sys/module/kvm_intel/parameters/nested 2>/dev/null; then
README.md:if [ "$CI" = true ]; then
integration/containerd/cri/integration-tests.sh:        if [ -n "$CI" ]; then
integration/openshift/run_openshift_tests.sh:if [ "$ID" != "fedora" ] && [ "$CI" == true ]; then
```
So, we have a mix of the two - but, I do think `true` is probably better - I just took the easy route... let me fix both of those things..